reframe/frontend/statistics.py

```diff
@@ -197,6 +199,41 @@ def json(self, force=False):
 
         return self._run_data
 
+    def junit(self, json_report, force=False):
+        # https://github.com/windyroad/JUnit-Schema/blob/master/JUnit.xsd
+        xml_testsuites = ET.Element('testsuites')
+        xml_testsuite = ET.SubElement(
+            xml_testsuites, 'testsuite',
+            attrib={
+                'name': 'rfm',
+                'errors': '0',
+                'failures': str(json_report['session_info']['num_failures']),
+                'tests': str(json_report['session_info']['num_cases']),
+                'time': str(json_report['session_info']['time_elapsed']),
```

Comment:
Does it make sense to add additional fields, like the `hostname` the reframe invocation command? Or other stuff?

Reply:
https://github.com/eth-cscs/reframe/pull/1925/commits/75f1bb4856604d0cda89e4eb9db259599e44ed25